lmdeploy/utils.py

```diff
@@ -128,7 +128,7 @@ def get_logger(
     formatter = ColorFormatter(log_formatter)
     for handler in handlers:
         handler.setFormatter(formatter)
-        handler.setLevel(log_level)
+        handler.setLevel(logging.DEBUG)
```

Comment:
why?

Reply:
To enable `--log-level DEBUG` for the script. The current default log level is `INFO`, if a user wants to set `DEBUG`, he will get `INFO` level log only since `DEBUG` is lower than `INFO`. So we have to set the default level `DEBUG` for the handler to enable all the level choices.